build.sbt

```diff
@@ -22,7 +22,7 @@ val scalaTestVersion = "2.2.4"
 val jodaVersion = "2.9"
 
 val sparkStreamingKafkaDep: String = {
-  if (dseVersion.toDouble >= 5.1) { 
+  if (sparkVersionArr(0).toInt > 2 || (sparkVersionArr(0).toInt == 2 && sparkVersionArr(1).toInt > 10)) {
```

Comment:
Do we mean use the scala version in the right-hand side of this comparison? Could we do something like: `|| dseScalaMajorMinorVersion.toDouble >= 2.10`